### ISSUE TITLE
Make radiation correction factor global across meshes

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#2353]
```
Currently, the radiation correction C, which scales the integrated radiation loss to
achieve the specified radiative fraction, is calculated on a mesh by mesh basis.  This
means that multi-mesh calculations of a fire plume will potentially give different
radiation patterns depending the mesh arrangement.  The factor C should be specified
globally, which would require a mesh exchange.  But, given all the time lags and angle
skipping we do in the radiation solver anyway, it seems we could come up with a scheme
that would give consistent results independent of the mesh arrangement.
```

Original issue reported on code.google.com by `randy.mcdermott` on 2015-04-01 18:56:38

**Comment by gforney** [comment]
```
(No text was entered with this change)
```

Original issue reported on code.google.com by `randy.mcdermott` on 2015-04-01 19:04:03
- **Labels added**: Milestone-FDS_6.3

**Comment by gforney** [comment]
```
This could be a problem. Many meshes may have little or no reactions taking place, in
which case C would be erratic. There would have to be some lower bound on the HRR to
make sure C is OK.
```

Original issue reported on code.google.com by `mcgratta` on 2015-04-01 19:35:09

**Comment by mcgratta** [comment]
Fixed #3707.
